.vscode/tasks.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+      {
+          "label": "MvcSandbox-build",
```

Comment:
Ditto here. Let's move this task to the `Mvc` directory.

Reply:
Removed this task entry!